### PR TITLE
fix: correctly display zoom in accelerator on mac

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Resolves https://github.com/electron/electron/issues/15496.

Backports [this CL](https://chromium-review.googlesource.com/c/chromium/src/+/1157224) from Chromium to fix `Cmd+Plus` to zoom keyboard shortcut. Tested to fix the display, but there is some weirdness around still needing `Shift` to trigger the correct zoom behavior. 

Ideally, i think we should enable both `Cmd+Plus` _and_ `Shift+Cmd+Plus` to zoom on MacOS, as this most closely mirrors corresponding behavior on Chrome itself.

/cc @brenca @alexeykuzmin @MarshallOfSound 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes

Notes: fix incorrect display of 'zoom in' accelerator on mac

### CODE REVIEW

File: patches/common/chromium/fix_zoom_display.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Web Oct 31 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Web, 31 Oct 2018 09:08:02 -0700
+Subject: fix_zoom_display.patch
```
Comment (MarshallOfSound) [inline patches/common/chromium/fix_zoom_display.patch]: Can we add a note here which Chromium version we can remove this patch for 👍

**Comment by alexeykuzmin** [comment]
> Tested to fix the display, but there is some weirdness around still needing Shift to trigger the correct zoom behavior.

This is sad =/

**Comment by codebytere** [comment]
@alexeykuzmin yeah that's why i marked it WIP

**Comment by alexeykuzmin** [comment]
@codebytere Let's merge this PR since it fixes the menu issue,
And create a new PR for the shortcut issue.

**Pull request merged**